### PR TITLE
Fixes to Sched Searches and Alerts to prep for AI/ML hub

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request edits Sched Searches and Alerts to prep for the [AI/ML hub page](https://github.com/SumoLogic/sumologic-documentation/pull/3793) release. Both sections were pretty outdated (not updated since ~2017).

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/alerts/scheduled-searches/create-email-alert.md [hunk 1]
```diff
@@ -28,15 +32,91 @@ Before creating an email alert, you'll first need to [create a scheduled search]
...
+
+### Alert email
+
+Set up Alert Conditions if you'd rather just receive an alert email when certain conditions are met. This means you'd get an automated alert when or if parameters you set are triggered, based on the results of the scheduled search. For example, you could set up an alert if a certain number of users visit the free trial URL of your website. You can [run a search from this email](#run-a-search-from-an-alert-email) when you receive it.
+
+You can be very specific with the alert condition — you can even set an exact number of results that triggers the email. Results can either be the number of log messages *OR* the number of aggregates returned by the saved search. If your saved search returns log messages, then the alert will use the number of messages you specify. If your query produces aggregate results, the alert will use the number of aggregates (or groups).
+
+#### Run a search from an alert email
+
+You can run a search directly from an Alert Email. When you receive an email created by an Email Alert in a Scheduled Search, only the first 25 results are displayed. (We do not want to flood your email with hundreds of thousands of search results.)
```
Comment (jpipkin1) [inline docs/alerts/scheduled-searches/create-email-alert.md:71]:
```suggestion
You can run a search directly from an alert email. When you receive an email created by an email alert in a Scheduled Search, only the first 25 results are displayed. (We do not want to flood your email with hundreds of thousands of search results.)
```

File: docs/alerts/scheduled-searches/create-email-alert.md [hunk 2]
```diff
@@ -28,15 +32,91 @@ Before creating an email alert, you'll first need to [create a scheduled search]
...
+
+### Real-time alerts
+
+:::note
+Not available to Sumo Logic Free accounts.
+:::
+
+Real Time Alerts continuously monitor your Sumo Logic deployment, and return alert emails whenever conditions are met. You can learn more in [Create a Real Time Alert](create-real-time-alert.md).
+
+Scheduled searches run according to the time zone of an individual's computer and browser, not according to the time zone of logs.
```
Comment (jpipkin1) [inline docs/alerts/scheduled-searches/create-email-alert.md:92]: Should you even include the "Real-time alerts" section if the feature is [deprecated](https://help.sumologic.com/docs/alerts/scheduled-searches/deprecation/)?

File: docs/alerts/scheduled-searches/create-email-alert.md [hunk 3]
```diff
@@ -28,15 +32,91 @@ Before creating an email alert, you'll first need to [create a scheduled search]
...
+
+## Customize your email alert subject and content
+
+You can use variables to customize the subject of your email. You can also select the features you want to include in your email. For details, see [Create an Email Alert.](create-email-alert.md)
```
Comment (jpipkin1) [inline docs/alerts/scheduled-searches/create-email-alert.md:96]:
```suggestion
You can use variables to customize the subject of your email. You can also select the features you want to include in your email. For details, see [Create a Scheduled Search Email Alert.](create-email-alert.md) 
```

File: docs/alerts/scheduled-searches/create-email-alert.md [hunk 4]
```diff
@@ -28,15 +32,91 @@ Before creating an email alert, you'll first need to [create a scheduled search]
...
+## Email alert template
+
+The Scheduled Search Email Alert template includes the following details:
+
+* **Saved Search.** Title of the saved search. 
+* **Description.** Description of the saved search. 
+* **Search String.** The search query of the saved search. 
+* **Time Range.** The time range that the saved search is set to use. 
+* **Run Frequency.** The frequency that the saved search is set to run. 
+* **Notification Threshold.** This is the alert condition that the saved search is set to use.
```
Comment (jpipkin1) [inline docs/alerts/scheduled-searches/create-email-alert.md:111]:
```suggestion
* **Notification Threshold.** The alert condition that the saved search is set to use.
```

File: docs/alerts/scheduled-searches/index.md [hunk 5]
```diff
@@ -1,13 +1,12 @@
 ---
 slug: /alerts/scheduled-searches
 title: Scheduled Searches
-sidebar_label: Scheduled Searches
 description: You can schedule log searches to send alerts.
 ---
```
Comment (jpipkin1) [inline docs/alerts/scheduled-searches/index.md:6]: You need to update the titles of the cards in this article to match the titles of the articles, and you need to put a few in the correct order.

File: docs/alerts/webhook-connections/servicenow/index.md [hunk 6]
```diff
@@ -6,48 +6,40 @@ description: With the ServiceNow integration, search results from Sumo Logic can
...
+To access [ServiceNow connections](/docs/alerts/webhook-connections/servicenow/), go to **Manage Data** > **Monitoring** > **Connections**. From here, you can search, edit, delete, and view of the details of Connections.
 
 In this section, we'll introduce the following concepts:
 
 <div className="box-wrapper">
 <div className="box smallbox card">
   <div className="container">
-  <a href="/docs/alerts/webhook-connections/servicenow/launch-investigations"><img src={useBaseUrl('img/icons/search.png')} alt="icon" width="40"/><h4>Launch Investigations</h4></a>
-  <p>Learn how to launch ad-hoc investigations directly from search results in Sumo Logic.</p>
+  <a href="/docs/alerts/webhook-connections/servicenow/set-up-security-incident-webhook"><img src={useBaseUrl('img/icons/security/security-capabilities.png')} alt="icon" width="40"/><h4>Set Up Security Incident Webhooks</h4></a>
```
Comment (jpipkin1) [inline docs/alerts/webhook-connections/servicenow/index.md:18]: Move this card to the end.

File: docs/alerts/scheduled-searches/schedule-search.md [hunk 7]
```diff
@@ -1,120 +1,114 @@
...
-sidebar_label: Get Started
-description: When you save a search, you can add a schedule to run it at a regularly scheduled time, and add alerts.
+title: Create a Scheduled Search
+description: Learn how to save and schedule a log search to run at a regularly scheduled time, and add alerts.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-When you save a search, you can add a schedule to run it at a regularly scheduled time, and add alerts. For alert types, see [Scheduled Searches](/docs/alerts/scheduled-searches). 
+This document outlines the process of creating a Scheduled Search, which is essentially a saved [Log Search](/docs/search) that you set to run at specified intervals and configure with [alerts](#alert-types).
```
Comment (jpipkin1) [inline docs/alerts/scheduled-searches/schedule-search.md:9]:
```suggestion
This article outlines the process of creating a Scheduled Search, which is essentially a saved [Log Search](/docs/search) that you set to run at specified intervals and configure with [alerts](#alert-types).
```

**Review by jpipkin1 — APPROVED** [review]
I found only a few minor things.

File: docs/alerts/scheduled-searches/index.md (hunk 5 above)
Comment (kimsauce) [reply docs/alerts/scheduled-searches/index.md:6]: good catch! will do this in my next PR push.

File: docs/alerts/scheduled-searches/create-email-alert.md (hunk 2 above)
Comment (kimsauce) [reply docs/alerts/scheduled-searches/create-email-alert.md:92]: Good call. For consistency with [this doc](https://help.sumologic.com/docs/alerts/scheduled-searches/create-real-time-alert/), I'll add the deprecation warning for now.

File: docs/alerts/webhook-connections/servicenow/index.md (hunk 6 above)
Comment (kimsauce) [reply docs/alerts/webhook-connections/servicenow/index.md:18]: This was by far the top-ranked doc in Google Analytics for this section - I think I'll leave this here

**Comment by kimsauce** [comment]
> Nice cleanup, @kimsauce! I found only a few minor things.

thank you for the thorough review!

**Pull request merged**